Branch-TestBed/Branch-SDK-Tests/BranchLoggerTests.m

```diff
@@ -11,99 +11,251 @@
 #import "Branch.h"
 
 @interface BranchLoggerTests : XCTestCase
+
 @end
 
 @implementation BranchLoggerTests
 
+// public API test
 - (void)testEnableLoggingSetsCorrectDefaultLevel {
     [[Branch getInstance] enableLogging];
     XCTAssertEqual([BranchLogger shared].logLevelThreshold, BranchLogLevelDebug, "Default log level should be Debug.");
 }
 
-- (void)testLogLevelThresholdBlocksLowerLevels {
+- (void)testLoggingEnabled_NOByDefault {
     BranchLogger *logger = [BranchLogger new];
-    logger.loggingEnabled = true;
-    logger.logLevelThreshold = BranchLogLevelDebug;
     
-    XCTestExpectation *expectation = [self expectationWithDescription:@"Log callback expectation for message that should pass the threshold"];
-
+    __block int count = 0;
```

Comment:
Is this thread safe? i.e when doing assertions, are there other tests that can run interference?